storagenode/notifications/notifications.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package notifications
+
+import (
+	"context"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// DB tells how application works with notifications database.
+//
+// architecture: Database
+type DB interface {
+	Insert(ctx context.Context, notification NewNotification) (Notification, error)
+	List(ctx context.Context, cursor NotificationCursor) (NotificationPage, error)
+	Read(ctx context.Context, notificationID uuid.UUID) error
+	ReadAll(ctx context.Context) error
+}
+
+// NotificationType is a numeric value of specific notification type.
+type NotificationType int
```

Comment:
Stutter. `notifications.NotificationType`, `notifcations.Type` would be sufficent. See